### PR TITLE
feat: add link for nuxt starter

### PR DESCRIPTION
**Pull request by danielroe**
This adds a link to the Nuxt starter repo. I've used a `/fork` link given your changes in https://github.com/stackblitz/webcontainer-core/pull/37, but it would be nice to keep it up-to-date with the GitHub repo it comes from if possible (https://github.com/nuxt/starter).

A URL like https://stackblitz.com/fork/github/nuxt/starter perhaps?

### CODE REVIEW

**Comment by EricSimons** [comment]
Oh perfect! We can just link to https://stackblitz.com/github/nuxt/starter 💪 

One thing- would it possible for y'all to include a `package-lock.json` file in that repo? That ensures that installs are super fast on every page load, otherwise we have to resolve a new lockfile for every user who opens up the project (which takes an additional 10-15 seconds)

**Comment by EricSimons** [comment]
One last thing- if you create a file in the root called `.stackblitzrc` with these values it will auto-install & boot devserver automatically on page load:

```
{
  "installDependencies": true,
  "startCommand": "npm dev"
}
```

**Comment by pi0** [comment]
Hi @EricSimons Thanks for the tips. Is it possible to link forking from specific branch? otherwise we can rename it to starter-stackblitz with lockfile and stackblitzrc

File: Supported_frameworks.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ The following is a list of frameworks that we have actively tested and are suppo
 
 - [Next.js](https://stackblitz.com/fork/nextjs)
 - [NestJS](https://stackblitz.com/fork/nestjs-starter)
+- [Nuxt](https://stackblitz.com/fork/nuxt-starter)
```
Comment (danielroe) [inline Supported_frameworks.md:7]:
```suggestion
- [Nuxt](https://stackblitz.com/github/nuxt/starter/tree/stackblitz)
```

**Comment by danielroe** [comment]
Thanks for the suggestions @EricSimons - all updated and very speedy 🚀

**Comment by EricSimons** [comment]
amazing thank you @danielroe!! merging now and will have our folks get a starter project for this on our homepage asap 👍

**Pull request merged**